.github/workflows/hierarchy.yml

```diff
@@ -135,23 +134,6 @@ jobs:
       isTypescriptSDK: ${{ needs.diff.outputs.isTypescriptSDK == 'true' }}
       isGraphQlTransport: ${{ needs.diff.outputs.isGraphQlTransport == 'true' }}
 
-  vercel-deploy:
```

Comment:
This is still in the confluence page, I can remove it myself but just wanted to check

Reply:
I missed this job, confluence updated